src/sentry/static/sentry/app/views/organizationDiscover/index.jsx

```diff
@@ -141,13 +142,15 @@ const OrganizationDiscoverContainer = createReactClass({
         </Body>
       </DiscoverContainer>
     );
-  },
+  }
 
   render() {
     const {isLoading, savedQuery, view} = this.state;
 
     const {location, params} = this.props;
-    const hasFeature = this.getFeatures().has('discover');
+
+    const {organization} = this.context;
+    const hasFeature = new Set(organization.features).has('discover');
```

Comment:
Is there a benefit for creating a set?

```suggestion
    const hasFeature = organization.features.includes('discover');
```

Reply:
Not much difference either way, but we do always access features as a set elsewhere